docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
       - ./server:/app/server
       - ./main.py:/app/main.py
       - ./config.py:/app/config.py
+    environment:
```

Comment:
We should set this in the Dockerfile to avoid having to specify this in the K8 conf